plugins/postcss-nesting/test/layer.css

```diff
@@ -0,0 +1,14 @@
+.container {
+	contain: layout inline-size;
+}
+
+.contained {
+	background-color: blue;
+	height: 100px;
+	width: 100px;
+
+	@layer A {
+		/* There is no definition for this behavior */
+		background-color: red;
+	}
```

Comment:
I have no idea if this should be allowed and how it should work exactly.

Mirroring `@media` and `@supports` handling seemed logical and should be ok for now.

_Not handling it here causes issues in the `cascade-layer` plugin if it encounters `@layer` in a style rule._

Reply:
That makes sense, I did a brief integration test and saw the issues!